backend/controller/controller.go

```diff
@@ -792,6 +796,28 @@ func (s *Service) reconcileDeployments(ctx context.Context) error {
 	return nil
 }
 
+func (s *Service) reconcileRunners(ctx context.Context) error {
+	log := log.FromContext(ctx)
+	log.Infof("Reconciling runners")
+
+	activeDeployments, err := s.dal.GetActiveDeployments(ctx)
+	if err != nil {
+		return errors.Wrap(err, "failed to get deployments needing reconciliation")
+	}
+
+	totalRunners := 0
+	for _, deployment := range activeDeployments {
+		totalRunners += deployment.MinReplicas
+	}
+
+	err = s.runnerScaling.SetReplicas(ctx, totalRunners)
```

Comment:
I don't think this covers the case of scaling down correctly?

Reply:
Def not. I need to add scaling down logic still